src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --main-bg-color: #e1e7ed;
+  --main-bg-gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+body {
+  margin-top: 20px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.input::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+}
+
+.input {
+  font-family: "Avenir", georgia, serif;
```

Comment:
add color property here and to placeholder

Reply:
I forgot to look color ))